aiohttp/abc.py

```diff
@@ -63,7 +64,9 @@ def handler(self) -> Callable[[Request], Awaitable[StreamResponse]]:
 
     @property
     @abstractmethod
-    def expect_handler(self) -> Callable[[Request], Awaitable[None]]:
+    def expect_handler(
+        self,
+    ) -> Callable[[Request], Awaitable[Union[None, StreamResponse]]]:
```

Comment:
This is based on the documentation: https://docs.aiohttp.org/en/stable/web_advanced.html?highlight=expect%20handler#expect-header